ambry-replication/src/main/java/com/github/ambry/replication/ReplicaThread.java

```diff
@@ -446,15 +602,13 @@ public void replicateCyclic() {
             remoteReplicaGroups.add(group);
           }
         }
-        if (standbyReplicasWithNoProgress.size() > 0) {
-          List<RemoteReplicaInfo> standbyReplicasTimedOutOnNoProgress =
-              getRemoteStandbyReplicasTimedOutOnNoProgress(standbyReplicasWithNoProgress);
```

Comment:
Curious to know the logic behind the removal of this part of code.

Reply:
Currrent logic is to first get standby replicas with no progress and filter the replicas which have timed out and create groups from them. 
I have made change to directly get stand by replicas which have timed out.